src/math/service/rem_pio2_large.rs

```diff
@@ -0,0 +1,489 @@
+use ::scalbn;
+use ::F64Ext;
```

Comment:
you can't use anything from outside the `math` module. You'll have to use free functions instead of methods.